### ISSUE TITLE
Make events.txt update optional

### ISSUE THREAD

**Issue by JaimeLeon2** [MobiFlight/MobiFlight-Connector#1045]
**Is your feature request related to a problem? Please describe.**
Updating events takes a relatively long time.  The events.txt usage in Mobiflight has been deprecated for about one year, yet updating events from Hubhop still updates these deprecated files.  In case someone still needs these files, I propose to create a new option in Settings to disable update of events.txt and related files.

**Describe the solution you'd like**
A new option in Settings menu to disable update of events.txt and related files.

**Describe alternatives you've considered**
Not doing anything.

**Additional context**
Update time can be significantly reduced.  It is currently a waste of time and hard disk space to keep updating these files.  Newer MF versions don't need them.  Older version users can continue using them with no impact to them.

**Comment by neilenns** [comment]
Is this even used by MobiFlight at all anymore? Why not jsut remove it entirely?

**Comment by JaimeLeon2** [comment]
I don't think they are needed at all in Mobiflight. I see no point in maintaining them. It is OK for Hubhop to generate the events.txt file in case some outside app wants to use it (like Fsuipc7).
I would agree they could be eliminated all together. 
In fact, some confused users are still trying to use them after watching old tutorials.

**Comment by DocMoebiuz** [comment]
We had decicded that we still keep them for users who created a config with the first version of the system and haven't update their config entries yet. 

Of course as long as their original events text resides in the wasm module the events will work. If an event was updated, the user will have to edit the configuration and that will also update it to the new presets system.

So yeah! It's probably fair to assume that there is no disadvantage for anyone if we remove the download altogether.

The big file is by the way the new presets file json. Download of the file takes way longer than the events.txt
